test/data/test_modules.py

```diff
@@ -0,0 +1,103 @@
+import torch
+from torchtext.modules import MultiheadAttentionContainer, \
+    ScaledDotProduct, MultiheadInProject, MultiheadOutProject
+from torch.nn.functional import multi_head_attention_forward as mha_forward
+from torch.testing import assert_allclose
+from ..common.torchtext_test_case import TorchtextTestCase
+
+
+class TestModels(TorchtextTestCase):
+
+    def test_multiheadattention(self):
+        embed_dim, nhead, tgt_len, src_len, bsz = 10, 5, 6, 10, 64
+        # Build torchtext MultiheadAttention module
+        MHA = MultiheadAttentionContainer((MultiheadInProject(embed_dim, nhead),
```

Comment:
Is MultiheadInProject really necessary? Why not just torch.nn.functional.linear? or torch.nn.Linear?